src/components/Popover/Popover.tsx

```diff
@@ -136,6 +138,7 @@ export class Popover extends React.Component<
         hasClose: false,
         initialOpen: false,
         disabled: false,
+        forceLinksAppearance: true,
```

Comment:
It is better to have names of properties when you don't need to set any values in defaultProps. What do you think about `disableLinksAppearance`?

Reply:
it looks like "inverted" prop for me, like having `notOpen` instead of `open`

Reply:
Inverted for "notOpen" is 'closed', your example with negation, and I agree that having negation in names is a bad pattern. But there is no negation in names like `disableAnyThing`, also there is standard property `disabled`.

Reply:
`disableLinksAppearance` will cause same negotiation logic


```js
if (!disableLinksAppearance) {
  //add modifier
}
```

Versus current


```js
if (forceLinksAppearance) {
  //add modifier
```

Reply:
Negation is bad as part of a name, but it's ok for conditions. More important to have intuitive set props.